config/norns.target

```diff
@@ -4,7 +4,6 @@ Description=norns
 Requires=sockets.target
 Requires=sound.target
 
-Requires=norns-init.service
```

Comment:
I'm OK with removing this, but I don't know the reasoning why @tehn added the set volume to 100% action in there.

If we remove this you can also remove the service file from the repo.